PerimeterXModule/DataContracts/Requests/RiskRequest.cs

```diff
@@ -14,6 +14,9 @@ public class RiskRequest
 		[DataMember(Name = "uuid", EmitDefaultValue = false)]
 		public string UUID;
 
+		[DataMember(Name = "firstParty", EmitDefaultValue = false)]
```

Comment:
We can keep the "convention" of calling this configuration as `firstPartyEnabled` (externally).

Reply:
i think the collector is looking for the value firstParty and not firstPartyMode
correct me if i'm wrong